apps/obs-web/src/views/Home.vue

```diff
@@ -88,9 +88,11 @@
         <br>
         <q-btn label="Declarations" to="/declarations" color="primary" exact style="margin: 5px"></q-btn>
 
-        <q-btn label="Trips" to="/trips" color="primary" exact style="margin: 5px"></q-btn>
+        <q-btn v-if="isAuthorized(['enforcement'])" label="EFP Management" to="/ole-efp-management" color="primary" exact style="margin: 5px"></q-btn>
 
-        <q-btn label="Logbook Capture" to="/log-book-capture" color="primary" exact style="margin: 5px"></q-btn>
+        <q-btn v-if="!isAuthorized(['enforcement'])" label="Trips" to="/trips" color="primary" exact style="margin: 5px"></q-btn>
```

Comment:
same comment as for Default.vue - rewrite to GRANT permission to appropriate roles.